src/Context/VideoContext.js

```diff
@@ -3,12 +3,16 @@ import { createContext, useContext, useReducer, useState } from "react";
 import { useEffect } from "react";
 import { videoReducer } from "../Reducer/Index";
 import { uniqueCategory, filterByCategory } from "../Utils/Index";
+import {postNotes} from "../Services/Index"
+import {useAuthContext} from "../Context/Index"
```

Comment:
You don't need to write the index in the path.

Reply:
Ok I'll remove it